benchmarks/urls.py

```diff
@@ -1,10 +1,9 @@
-from django.conf.urls import include, patterns, url
+from django.urls import include, path
 
 from .drest import router as drest_router
 from .drf import router as drf_router
 
-urlpatterns = patterns(
-    '',
-    url(r'^', include(drf_router.urls)),
-    url(r'^', include(drest_router.urls)),
-)
+urlpatterns = [
+    path('', include(drf_router.urls)),
```

Comment:
It's more modern/recommended in django versions > 2: https://stackoverflow.com/questions/55285814/when-should-i-use-path-over-re-path

And since we don't have to use a regex here, I thought that path made sense.